### ISSUE TITLE
[Accessibility] [MacCatalyst] Empty Entry inside SfTextInputLayout cannot be focused with Tab key

### ISSUE THREAD

**Issue by bartlomiejsawarski** [syncfusion/maui-toolkit#178]
### Description

According to [this WWDC video](https://developer.apple.com/videos/play/wwdc2021/10260/) text inputs (`UITextField` here) should be navigable using keyboard Tab key.

It's working correctly for [MAUI Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-9.0), but as soon as we place `Entry` inside `SfTextInputLayout` it stops working as long as the `Entry` is empty

### Steps to Reproduce

1. [Add syncfusion toolkit to empty MAUI project](https://github.com/syncfusion/maui-toolkit?tab=readme-ov-file#configure-syncfusion-toolkit)
2. In `MainPage.xaml` just add below code to `StackLayout`

```
            <textInputLayout:SfTextInputLayout
                Hint="Entry inside SfTextInputLayout"
                HorizontalOptions="Fill">
                <Entry />
            </textInputLayout:SfTextInputLayout>

            <Entry
                Placeholder="Entry outside SfTextInputLayout"/>
```
3. As you can see on attached video:
- first I click Tab key to set focus on entry outside SfTextInputLayout
- then I click Shift+Tab to prove that I cannot go back to first entry
- then I click with mouse on entry inside SfTextInputLayout, type some text
- then I click Tab, then Shift+Tab to move between both entries

https://github.com/user-attachments/assets/6505fddf-6588-4ce5-9e35-ead6bf7ea3a2

### Version with bug

1.0.4

### Is this a regression from previous behavior?

Not sure, haven't tested other versions

### Last Known Working Version

1.0.3

### Affected platforms

macOS

### Have you found a workaround?

No, but maybe some modifications on `UITextField` object inside `SfTextInputLayout` affect this unwanted behaviour

### Relevant log output

```shell

```
